entities/library.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require './entities/book'
+require './entities/reader'
+require './entities/author'
+require './entities/order.rb'
+require './seed'
+
+# library information
+class Library
+  attr_reader :readers, :orders, :authors, :books
+
+  include SeedData
```

Comment:
I think it should be called `Statistics` module

Reply:
done